org.mintguide.WebAppReader.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: org.mintguide.WebAppReader
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: /app/webappreader.sh
+finish-args:
+  - --device=dri
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
```

Comment:
Why is this needed?

Reply:
- --device=dri the program has a built-in browser, I hope that it works smoother thanks to this setting, if I'm wrong, I'll remove it.
- --filesystem=home this is for the possibility of saving pages to the hard disk

Reply:
Just to clarify `--device=dri` was NEVER the question here.
Not sure how it look so ambiguous:

![image](https://github.com/flathub/flathub/assets/114441/d1a6b9f5-d8c2-483f-bb26-982e721ab903)